cmd/license-header-checker/main.go

```diff
@@ -46,7 +46,8 @@ func main() {
 func getLicense(path string) string {
 	data, err := ioutil.ReadFile(path)
 	if err != nil {
-		panic(err)
+		log.Fatal(err)
```

Comment:
log.Fatal is equivalent to Print() followed by a call to os.Exit(1)
And if one file fails, you will stop the  process?? in this case only a few files will be updated, maybe is better to skip the fail and log it

Reply:
It is not "one file" it is **the license** that should be added to all the files ;)

Reply:
ok, it is the global licence, so.. u only need to remove the next **os.Exit(0)** because **log.Fatal(..)** already do it but with error code **1**

Reply:
internal code inside _log.go_

```
// Fatal is equivalent to Print() followed by a call to os.Exit(1).
func Fatal(v ...interface{}) {
	std.Output(2, fmt.Sprint(v...))
	os.Exit(1)
}
```

Reply:
Thanks